tests/utils.test.ts

```diff
@@ -27,6 +27,14 @@ test.describe("runProcess", () => {
   });
 });
 
+test("maps_mandates", async () => {
```

Comment:
Ditto on this change. What is it from?

Tip when opening a pull request: review your own code. Check what code has changed and make sure it is all intentional and the best code you can write.